packages/fxa-auth-server/test/local/redis.js

```diff
@@ -32,7 +32,7 @@ describe('lib/redis:', () => {
     };
     fxaShared = sinon.spy(() => redis);
     wrapper = proxyquire(`${LIB_DIR}/redis`, {
-      '../../../fxa-shared/redis': fxaShared,
+      '../../fxa-shared/redis': fxaShared,
```

Comment:
I'm not really sure why this change was necessary, I haven't looked to see if node 12 resolves modules differently to node 10.